### DISCUSSION TITLE
Error 404 when try install package

### DISCUSSION THREAD

**Discussion by Icammarano** [teambit/bit#5713] [Help]
Hello, I wanted to ask you why I get a 404 error when I try to install a component from my scope?

`[root@centos8-dev-03 prueba_button]# npm i @bit/xxx.xxx.button
npm ERR! code E404
npm ERR! 404 Not Found - GET https://node.bit.dev/@bit%2fxxx.xxx.button - Not found
npm ERR! 404
npm ERR! 404  '@bit/xxx.xxx.button@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.`

xxx is not the real scope, i replace it for ask.
I try to modify the .npmrc file but the result was the same

Thx for help.

**Comment by itaymendel** [comment]
Sadly the link you hid can tell a lot about the issue you experience.
Also, does this happens from your local or remote?

But if I'm forced to shoot from the hip here, I'd suggest running `bit login` and ensure that your `.npmrc` knows to fetch and authenticate with the `node.bit.cloud` remote registry.

Run `npm config ls` from the working directory of that workspace, to see exactly how npm resolves the configuration.

**Reply by Icammarano** [reply]
Hi itay, thx for your answer. If i execute npm config ls in the directory where i want to install the component, this is shown, after bit login:
```
[root@centos8-dev-03 prueba_button]# npm config ls
; cli configs
metrics-registry = "https://registry.npmjs.org/"
scope = ""
user-agent = "npm/6.14.8 node/v14.9.0 linux x64"

; userconfig /root/.npmrc
@bit:registry = "https://node.bit.dev"

; node bin location = /root/.nvm/versions/node/v14.9.0/bin/node
; cwd = /opt/button_bit_prueba/prueba_button
; HOME = /root
``` 

I remove the .npmrc file because it doesn't work either. Or could you tell me how I have to configure that file so that I can download the package? it's all remote.

Thanks you so much
